pkg/maker/buildcontent.go

```diff
@@ -914,3 +914,12 @@ func (m *Maker) ExecutesCommands(executes []Executes) string {
 	}
 	return content
 }
+
+func (c *Cbuild) GetAPIFiles(id string) []Files {
+	for _, api := range c.BuildDescType.Apis {
+		if api.API == id {
+			return api.Files
+		}
+	}
+	return []Files{}
```

Comment:
Is missing files is an error case?
If yes, consider returning an error or a bool indicating success.

Reply:
An empty container is a normal use case, indeed not every component implements an API.
Returning `nil` is also fine, I will commit your suggestion.